.github/workflows/nightly.yml

```diff
@@ -7,26 +7,68 @@ on:
     inputs:
       ignore_commits_force_release:
         type: boolean
-        description: 'Ignore (new) commits and force a release'
+        description: 'Do not check for new commits / Force new release'
         default: false
 
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
 jobs:
   release:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: '0'
```

Comment:
You probably want to move this to the NewPipe clone

Reply:
I assume I should move the ``permissions`` to ``jobs.release`` or? (although it really doesn't make that kind of a big difference since there is only one job)

Anyway should also be fixed now ^^

Reply:
No I mean `force-depth`, it should be on the NewPipe clone, not on the nightly repo clone

Reply:
To quote from [the docs](https://github.com/actions/checkout):

``fetch-depth``
> Number of commits to fetch. 0 indicates all history for all branches and tags.
> Default: 1

So the default value of ``1`` should be fine.

~I should probably also change it the defaults on this (the nightly) repo~

Update: Nevermind bad idea, then we have no tags and we try to re-release ``nightly-1`` which causes a crash

Reply:
Oh nevermind I thought `fetch-depth=0` meant "shallow clone", while it actually means the opposite.